db/migrate/20231026033633_create_like.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLike < ActiveRecord::Migration[7.1]
+  def change
+    create_table :likes do |t|
+      t.references :users,foreign_key:true
+      t.references :posts,foreign_key:true
```

Comment:
- [ ] Same here, The foreign keys between users table and likes table is `user_id`, not `users_id` and posts table and likes table is `post_id`, not `posts_id`. Please fix that.

[Important note] After you edit the migration files with the changes I suggested, Kindly run this command `rails db:migrate` to update the schema file.